src/main/java/InvalidCommandException.java

```diff
@@ -0,0 +1,10 @@
+public class InvalidCommandException extends DukeException{
+    public InvalidCommandException(String message) {
+        super(message);
+    }
+
+    @Override
+    public String toString() {
+        return "Uhh... You're gonna have to say that again, Red.";
+    }
```

Comment:
When you created this exception, you input a blank string. Instead, you should consider making the constructor take in no arguments, and super("Uhh.. You're gonna...").

Then to get the message, call getMessage().